client/api/omni/specs/siderolink.proto

```diff
@@ -28,6 +28,17 @@ message SiderolinkSpec {
   // RemoteAddr is the machine address how it's visible from Omni
   // it is determined by reading X-Forwarded-For header coming from the gRPC API.
   string remote_addr = 8;
+  // NodeUniqueToken is the per node join token which is saved in the Node META partition after
+  // the machine is accepted in Omni.
+  // Only for Talos >= 1.6.
+  string node_unique_token = 9;
+}
+
+// LinkStatusSpec is created when the link peer event was submitted.
+message LinkStatusSpec {
+  string node_subnet = 1;
+  string node_public_key = 2;
+  string virtual_addrport = 3;
```

Comment:
Why virtual?

Reply:
that's for the wireguard over gRPC. We keep it there to keep track of it being updated.